### PR TITLE
Changelogs and Readme files.

### PR DESCRIPTION
**Pull request by zie1ony**
Top level README.md is still under construction and will be finished in further PRs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,77 @@
...
+        contract.flip();
+        assert_eq!(contract.get(), true);
+    }
+}
 ```
 
-## Modes
+## Backends
+
+Odra is designed to integrate any most WebAssemby-based smart contract systems.
```
Comment (kubaplas) [inline README.md:57]: Any or most?

File: core/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Odra Core
+
+Home of the `odra` crate. It glues together all other Odra crates and provides easy to use interace.
```
Comment (kubaplas) [inline core/README.md:3]: interface

File: core/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+It has 3 modes controlled using features. Only one feature at the time should be used:
+- `mock-vm`, enabled by default, used to run the code against `MockVM`.
+- `wasm`, should be used when compiling code to WebAssembly.
+- `wasm-test`, used to run the code agains a provided backend virtual machine.
```
Comment (kubaplas) [inline core/README.md:8]: against

File: lang/ir/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Odra Lang IR (Intermediate Representation)
+
+`odra-ir` parses code into structs that later can by `odra-codegen`.
```
Comment (kubaplas) [inline lang/ir/README.md:3]: later can by what?

File: utils/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+Changelog for `odra-utils`.
+
+## [0.0.1] - 2022-07-20
+### Added
+- `camel_to_snake` function for the contract name conversion.
+- `event_absolute_position`function to calculate the event absolute position.
```
Comment (kubaplas) [inline utils/CHANGELOG.md:8]: space

**Review by kubaplas — CHANGES_REQUESTED** [review]
:+1:

File: README.md (hunk 1 above)
Comment (zie1ony) [reply README.md:57]: Most, because if smart contract system is not account-based like Bitcoin, it has totally different concepts.

**Review by kubaplas — APPROVED** [review]

**Pull request merged**